workbench-core/authorization/src/dynamicAuthorization/dynamicAuthorizationService.test.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { AuthenticatedUser } from '../authenticatedUser';
+import { CreateGroupResponse } from './dynamicAuthorizationInputs/createGroup';
+import { DynamicAuthorizationService } from './dynamicAuthorizationService';
+import { GroupManagementPlugin } from './groupManagementPlugin';
+
+describe('WBCGroupManagemntPlugin', () => {
+  const mockUser: AuthenticatedUser = {
```

Comment:
not blocking, but Jest strongly recommends initializing all code inside `beforeEach/All`s because of potential async and overloading issues.

Reply:
fixed!